packages/yoga/src/Result/native/Tags.jsx

```diff
@@ -25,7 +25,7 @@ const Wrapper = styled.ScrollView`
 const ResultTags = ({ items }) => (
   <Wrapper horizontal showsHorizontalScrollIndicator={false}>
     {items.map(({ ...props }) => (
```

Comment:
What is returned in that `...props`?

Don't exist any unique value to use as key in `<TagStyled />?

Reply:
We can use the `variation` value as key, since that we have only one variant in component. right?

Reply:
cc/ @alycecristines